app/views/users/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Single user page</h1>
\ No newline at end of file
+<%= render partial: 'partials/user_card', locals: { user: @user } %>
+
+<div class='bio'>
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class='feed'>
+  <% @user.recent_posts.each do |post| %>
+    <%= render partial: 'partials/post', locals: { post: post, short: true } %>
+  <% end %>
+  <% if @user.posts.length > 0 %>
+    <%= link_to "/users/#{@user.id}/posts" do %>
+      <button type='button'>See all posts</button>
+    <% end %>
```

Comment:
Can you populate your database to See all post as it is displayed on the 👉 [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). Because the requirement is asking to displayed all posts